index.md

```diff
@@ -21,6 +33,10 @@ Paragraph of introductory material.
 
 ## Other Resources
 
+*   [Installation instructions](install.html)
 *   [Reference](reference.html)
 *   [Discussion](discussion.html)
 *   [Instructor's Guide](instructors.html)
+
+[swc-workshop-template]: http://swcarpentry.github.io/workshop-template/index.html
```

Comment:
This still needs an HTTPS link (suggesting students follow installation instructions retrieved over HTTP is not secure).